src/version.rs

```diff
@@ -68,9 +71,90 @@ pub struct VersionLinks {
     pub dependencies: String,
     pub version_downloads: String,
     pub authors: String,
+    pub build_info: String,
+}
+
+#[derive(Insertable, AsChangeset)]
+#[table_name="build_info"]
+#[primary_key(version_id, rust_version, target)]
+struct NewBuildInfo {
```

Comment:
If we expect this table to be relatively stable, it might be worth just dumping this struct and adding `#[derive(Insertable, AsChangeset)]` onto `BuildInfo`.